.github/workflows/ubuntu_test.yml

```diff
@@ -0,0 +1,91 @@
+name: UbuntuSimple
+
+on:
+  push:
+    branches:
+      - master
+      - 'dev**'
+  pull_request:
+    paths:
+      - 'tesseract**'
+      - '.github/workflows/ubuntu.yml'
+      - '**.repos'
+  schedule:
+    - cron: '0 5 * * *'
+  release:
+    types:
+      - released
+
+jobs:
+  ci:
+    name: ${{ matrix.distro }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        distro: [focal, jammy]
+    container:
+      image: ubuntu:${{ matrix.distro }}
+      env:
+        CCACHE_DIR: "$GITHUB_WORKSPACE/${{ matrix.distro }}/.ccache"
+        DEBIAN_FRONTEND: noninteractive
+        TZ: Etc/UTC
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          path: target_ws/src
+
+      - name: Install Depends
+        shell: bash
+        run: |
+          apt update
+          apt upgrade -y
+          apt install -y cmake curl lsb-release ca-certificates gnupg python3 python3-distutils python3-pip ccache git
+          python3 -m pip install vcstool -q
+          python3 -m pip install colcon-common-extensions -q
+          cd $GITHUB_WORKSPACE/target_ws/src
+          ./.github/workflows/add_ros_apt_sources.sh
+          apt install -y liboctomap-dev libfcl-dev libconsole-bridge-dev libbullet-dev libbullet-extras-dev libyaml-cpp-dev libtinyxml2-dev libassimp-dev liborocos-kdl-dev libpcl-dev liblapack-dev libboost-dev libboost-filesystem-dev libboost-serialization-dev libboost-program-options-dev libboost-graph-dev libbenchmark-dev libccd-dev libgtest-dev
```

Comment:
Any reason to not use `rosdep` here? Seems like it would be a lot more convenient and less error-prone

Reply:
It looks like rodep is a python tool so that should be fine.